nima/http2/webclient/src/main/java/io/helidon/nima/http2/webclient/ClientRequestImpl.java

```diff
@@ -217,7 +218,13 @@ private Http2Headers prepareHeaders(WritableHeaders<?> headers) {
 
     private Http2ClientStream reserveStream() {
         if (explicitConnection == null) {
-            ConnectionKey connectionKey = new ConnectionKey(uri.scheme(), uri.authority(), uri.host(), uri.port(), tls);
+            ConnectionKey connectionKey = new ConnectionKey(uri.scheme(),
+                                                            uri.authority(),
```

Comment:
`authority` is a combination of `host` and `port`, we should not use both at the same time.

Reply:
fixed